src/DGtal/dec/doc/moduleDECEmbedding.dox

```diff
@@ -0,0 +1,130 @@
+/**
+ * @file moduleDECEmbedding.dox
+ * @author David Coeurjolly (\c david.coeurjolly@liris.cnrs.fr )
+ * @author Pierre Gueth (\c pierre.gueth@liris.cnrs.fr )
+ * Laboratoire d'InfoRmatique en Image et Systemes d'information - LIRIS (CNRS, UMR 5205), CNRS, France
+ *
+ * @date 2014/04/04
+ *
+ * Documentation file for feature DECEmbedding
+ *
+ * This file is part of the DGtal library.
+ */
+
+/*
+ * Useful to avoid writing DGtal:: in front of every class.
+ * Do not forget to add an entry in src/DGtal/base/Config.h.in !
+ */
+namespace DGtal {
+//----------------------------------------
+/*!
+@page moduleDECEmbedding Embedding
+@writers Pierre Gueth
+
+[TOC]
+
+\section sectDECEmbedding Embedding
+
+The DEC package allows the embedding of a discrete manifold contained in an higher dimension euclidean space.
+For example, one could use the DEC package to represent a curved surface embedded in a three dimensional euclidean space.
+Throughout this documentation n stands for in embedded manifold dimension,
+m stands for this ambient space dimension and
+k is used to characterize the dimension of cells inside the DEC structure.
+
+\f[ 0 \le k \le n \le m \f]
+
+Only lower dimensions k-cells can be inserted inside a DEC structure (k <= n).
+Those embedded and ambient dimensions are enforced as the first and second template parameters of DiscreteExteriorCalculus.
+Dual and primal operators are defined with respect the the embedding dimension.
+
+\section sectDECFactory DEC factory
+
```

Comment:
BTW, would it be possible to add an example to create a DEC from DigitalSurface.begin() and DigitalSurface.end() ?